storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc

```diff
@@ -20,6 +17,8 @@ This eliminates the need for cluster administrators to pre-provision storage.
 
 * The _CIFS/SMB CSI driver_ enables you to create and mount CIFS/SMB PVs.
 
+The CIFS/SMB CSI Driver Operator does _not_ support Windows Distributed File System (DFS).
```

Comment:
@lpettyjo I think we'd better move it to `Limitations ` section, also for the `no Kerberos auth support` and `only support Windows SMB server versions range`, wdyt?

Reply:
You are so right! I've changed the content accordingly.